ansible/config_sonic_basedon_testbed.yml

```diff
@@ -77,22 +77,17 @@
             regexp: '^enabled='
             line: 'enabled=false'
         become: true
- 
-      # reload the device and wait it to come back
-      - name: Reboot is required for minigraph change
-        shell: sleep 2 && shutdown -r now "Ansible Create new configuration Minigraph file, triggered reboot."
-        async: 1
-        poll: 0
+
+      - name: pip install pexpect to support ansible expect module 
```

Comment:
we could update the sonic-buildimage to install this module by default inside the docker.

Reply:
You are right, will add this module into sonic-buildimage by default inside the docker.